.storybook/polaris-readme-loader.js

```diff
@@ -71,6 +71,7 @@ ${example.storyName}.parameters = {
     minHeight: '720px',
     borderBottom: '1px solid #000',
     marginBottom: '8px',
+    padding: '10px',
```

Comment:
Looks like you're adding the padding here. Does this not show up if you're looking at it in the fullscreen layout?

Reply:
Moving to `fullscreen` removed the padding on the "All" component pages. So I added it back here.

Reply:
Gotcha. Is the intent to run all previews on fullscreen now? If so, then I think I like your solution, and I'd get rid of the `hasFullscreenLayout` conditional variable. Otherwise, if it's on a per-component basis, then updating the place Ben mentioned makes more sense